### ISSUE TITLE
Equal-weight small capitals.

### ISSUE THREAD

**Issue by xorgy** [xorgy/twenty-first#1]
Small capitals are common in documents set in Century, such as briefs submitted to the Supreme Court of the United States, but most (all?) widely available versions of Century do not have proper small capitals, so usually ordinary capitals are simply shrunk in both dimensions to set as small capitals.

An improvement on this would be to adjust those scaled capitals to have the same weight as full capitals. The shape and aspect ratio of small capitals in documents set with Century is usually the same as the full capitals, so probably that should not change.

**Comment by dumpyfrogg** [comment]
I agree! I _think_ TexGyreSchola (TeX version of C059) has true small capitals, but the line thicknesses match those of the lowercase, not capital letters. I don't know if that's a design choice or that it might've been a duct tape solution, because some fonts do have true small capitals with lowercase thicknesses.

I wonder if it's possible to adjust the thicknesses to match the capital letters with fontforge?

**Comment by xorgy** [comment]
> I wonder if it's possible to adjust the thicknesses to match the capital letters with fontforge?

There is probably some partially-automated way to do this... I didn't realize I would be re-nerdsniped by my own issue here.
